### ISSUE TITLE
Problem in CouchAdmin client creation

### ISSUE THREAD

**Issue by EcoFreak** [PHP-on-Couch/PHP-on-Couch#69]
I started to use PHP-on-Couch and have been using to create, delete and updated documents without problems. I'm trying to use CouchAdmin class to delete a user and are having some issues.

I'm adding extra options for curl in order to avoid using the my company's proxy configuration, like so:

 ```
  $couchDbClient = new CouchClient(env('COUCH_DB_SERVER', 'http://localhost:5984/'),
                'my_db',
                [
                    'username' => env('COUCH_DB_USER'),
                    'password' => env('COUCH_DB_PASSWORD'),
                    'curl' => ['CURLOPT_HTTPPROXYTUNNEL' => 0, 'CURLOPT_PROXY' => null]
                ]);
$admin = new CouchAdmin($couchDbClient);
```
I am able to create users, like so:
`$admin->createUser('user');`

Deleting users results in 'no data to parse'
`$admin->deleteUser('user');`

This leads me to think that the client options is not being used in the deleteUser method.

Any workarounds or is this a know bug?

**Comment by popojargo** [comment]
@EcoFreak I tested with the latest version from master and I wasn't able to reproduce the error.

I also published version 2.0.8 that fix a problem with line reading but I don't think it's related.

Maybe you could try to debug what happens when the $admin->deleteUser('user') is called?

**Comment by EcoFreak** [comment]
@popojargo I'm using the latest version.
I've just browsed the source code and found think I found the issue.

In the CouchAdmin's **createUser** method, the client to the users database is created like this:

`$client = new CouchClient($this->client->dsn(), $this->usersdb, $this->client->options());` _Line 235_

It creates the CouchClient passing the options of the client provided when constructing the CouchAdmin instance. In the **deleteUser** method the client is created but the options are not provided, like so:

`$client = new CouchClient($this->client->dsn(), $this->usersdb);` _Line 252_

This also happens in the **deleteAdmin** method.

Is it possible to fix this? I could do it but tbh I don't have much experience with Pull Requests and that sort of stuff :) (willing to learn dough)

Cheers

**Comment by popojargo** [comment]
If you're willing to learn, I guess you could try to fix it. 

I can directly support you via Gitter (Live chatting with github) available here: https://gitter.im/PHP-on-Couch/PHP-on-Couch

**Comment by popojargo** [comment]
Fixed in 2.0.9

**Comment by EcoFreak** [comment]
Thanks @popojargo
